spec/conjurctl/configuration_show_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+require 'open3'
+
+describe "conjurctl configuration show" do
+  it "prints the configuration in JSON format" do
+    stdout, _stderr , _status = Open3.capture3(
+      "conjurctl configuration show --output json"
+    )
+
+    expect(stdout).to eq(<<~JSON)
```

Comment:
We may want to do `include` matchers here if possible so that we don't have to update these tests every time we add a field.